src/ts/components/button.ts

```diff
@@ -22,12 +23,18 @@ export interface ButtonConfig extends ComponentConfig {
    * Default: false
    */
   acceptsTouchWithUiHidden?: boolean;
+
+  /**
+   * 
```

Comment:
Please add a short description

Reply:
I did add that but didn't push 🤦 it's there now in 604c91ff